src/utils/auth.ts

```diff
@@ -0,0 +1,40 @@
+import { jwtDecode, JwtPayload } from "jwt-decode";
+
+interface ExtendedJwtPayload extends JwtPayload {
+  email: string;
+}
+
+export const getTokenFromLocalStorage = () => {
+  // const token = localStorage.getItem("token") || '';
```

Comment:
Remove OR and check if the token does not exist. If it does not exist return null